src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/EnumWithVariantOfUint64.java

```diff
@@ -33,7 +33,7 @@
 
     static
     {
-        Long2ObjectHashMap<EnumWithVariantOfUint64> valueByLong = new Long2ObjectHashMap<>(5, 1.0f);
+        Long2ObjectHashMap<EnumWithVariantOfUint64> valueByLong = new Long2ObjectHashMap<>(5, 0.9f);
```

Comment:
The range for the load factor is 0.1 to 0.9. Is `0.9f` okay?

Reply:
Ah, yes - they are not anticipating the exact size initialization case, okay to go with `0.9f` instead.